lib/account/widgets/avatar.dart

```diff
@@ -12,36 +12,40 @@ class Avatar extends StatefulWidget {
     this.actionIcon = const Icon(Icons.photo_library),
     this.onAction,
     this.size,
+    this.withHero = false,
   });
 
   final bool isTappable;
   final Profile profile;
   final Icon? actionIcon;
   final VoidCallback? onAction;
   final double? size;
+  final bool withHero;
 
   @override
   AvatarState createState() => AvatarState();
 }
 
 class AvatarState extends State<Avatar> {
+  Widget buildAvatar() {
+    Widget avatar = CircleAvatar(
+      radius: widget.size,
+      backgroundImage: widget.profile.avatarUrl?.isEmpty ?? true ? null : NetworkImage(widget.profile.avatarUrl!),
+      child: widget.profile.avatarUrl?.isEmpty ?? true
+          ? Text(
+              widget.profile.username[0].toUpperCase(),
+              style: TextStyle(fontSize: widget.size),
+            )
+          : null,
+    );
+    return widget.withHero ? Hero(tag: "Avatar-${widget.profile.id}", child: avatar) : avatar;
```

Comment:
quick question because that is hard to see from the diff: Which avatars do not need the hero?